### PR TITLE
Update company name, emails, and site links

### PR DESCRIPTION
**Pull request by pietv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,43 +28,42 @@ See below for the directory structure created by `init`.
...
-* an ethereum node
-* a tendermint-testchain node
-* a tinydns daemon
+* a PGP daemon
+* an IPFS node
+* a Bitcoin node
+* a Bitcoin node with its rpc on
+* a Bitcoin TestNet node with its RPC on
+* an Ethereum node
+* a Tendermint test chain node
```
Comment (zramsay) [inline README.md]: let's remove the latter two bitcoin options, and the tendermint option. we can add the bigchainDB service, maybe zcash too?

File: cmd/eris.go [hunk 2]
```diff
@@ -26,10 +26,9 @@ const dmVerMin = version.DM_VER_MIN
 var ErisCmd = &cobra.Command{
 	Use:   "eris COMMAND [FLAG ...]",
 	Short: "The Ecosystem Application Platform",
-	Long: `Eris is a platform for building, testing, maintaining, and operating
-applications built to run on an ecosystem level.
+	Long: `Eris is an application platform for building, testing, maintaining, and operating applications built to run on an ecosystem level.
 
-Made with <3 by Eris Industries.
+Made with <3 by Monax Industries.
```
Comment (zramsay) [inline cmd/eris.go]: :(

File: initialize/default_keys.go [hunk 3]
```diff
@@ -30,12 +30,11 @@ status = "unfit for production"
 image = "` + path.Join(version.DefaultRegistry, version.ImageKeys) + `"
 data_container = true
 exec_host = "ERIS_KEYS_HOST"
-#restart = "always"
 
 [location]
 dockerfile = "https://github.com/eris-ltd/eris-keys/blob/master/Dockerfile"
-#see also http://github.com/eris-ltd/common/blob/master/docker/keys/build.sh"
+# See also http://github.com/eris-ltd/common/blob/master/docker/keys/build.sh
```
Comment (zramsay) [inline initialize/default_keys.go]: ^ link deprecated. we can actually remove that line.

File: tests/README.md [hunk 4]
```diff
@@ -7,25 +7,20 @@ This is a hard tool to test. In order to clearly define (and limit) the testing
...
-  * debian (not implemented. is this necessary?)
-  * rhel (not implemented. is this necessary?)
-  * ubuntu (running the tests in docker provides natively)
-  * OSX (not implemented. is this necessary?)
-  * windows (not implemented. is this necessary?)
+  * Debian (not implemented)
+  * RHEL/Centos (not implemented)
+  * Ubuntu (running the tests in Docker provides natively)
+  * OSX (not implemented)
+  * Windows (not implemented)
```
Comment (zramsay) [inline tests/README.md]: aren't testing for macOS and Windows imiplemented...?

Comment (zramsay) [inline tests/README.md]: oiii, this file either needs to be removed or moved and updated. Preferably the latter

**Comment by zramsay** [comment]
LGTM other than couple comments.

File: tests/README.md (hunk 4 above)
Comment (pietv) [inline tests/README.md]: Yeah, good spot! Also "macOS" instead of "OSX".

File: initialize/default_keys.go (hunk 3 above)
Comment (pietv) [inline initialize/default_keys.go]: Done

**Pull request merged**